src/x/instrument/config_prometheus.go

```diff
@@ -167,7 +169,7 @@ func (c PrometheusConfiguration) NewReporter(
 
 	handler := reporter.HTTPHandler()
 	if configOpts.Registry != nil {
-		gatherer := newMultiGatherer(configOpts.Registry, configOpts.ExternalRegistries)
+		gatherer := newMultiGatherer(configOpts.Registry, configOpts.ExternalRegistries, configOpts.CommonLabels)
```

Comment:
Hm, is there a way we can always append common labels even when `configOpts.Registry == nil`?

Perhaps we should just always be using a multi-gatherer that works when only the default registry?

i.e.

```go
// at top of this method:
registry := configOpts.Registry
if registry != nil {
  registry = prom.DefaultRegistry
}

// other... (replace usages of configOpts.Registry)

// now at this part of the method (no if statement):
gatherer := newMultiGatherer(registry, configOpts.ExternalRegistries, configOpts.CommonLabels)
handler := promhttp.HandlerFor(gatherer, promhttp.HandlerOpts{})
```

That way regardless of whether you pass in a registry or not via `configOpts.Registry` you always get the common labels if they are set.